Package.swift

```diff
@@ -10,7 +10,8 @@ let package = Package(
   dependencies: [
     // Support for dependabot for swift packages in dependabot
     // is in draft mode https://github.com/dependabot/dependabot-core/pull/3772
-    .package(url: "https://github.com/realm/SwiftLint.git", from: "0.43.1")
+    .package(url: "https://github.com/realm/SwiftLint.git", from: "0.43.1"),
+    .package(url: "https://github.com/meilisearch/meilisearch-swift.git", from: "0.12.0")
```

Comment:
Why did you add this line?

Reply:
That was my mistake sorry! I was trying to add the package for meiliisearch. I read the guidelines wrong. I just deleted that line a moment ago, while trying to fix this.